src/Http/Exceptions/Handler.php

```diff
@@ -23,6 +25,12 @@ public function render($request, Throwable $exception)
                 'message' => $exception->getMessage(),
             ], 422);
         }
+        elseif ($exception instanceof DirectoryOutsideOfRootException)
+        {
+            return new JsonResponse([
+                'message' => $exception->getMessage(),
+            ], 405);
+        }
         else
```

Comment:
If each conditional statement evaluates to return, the last else is not needed.

Reply:
Right, it's rather an aesthetics that leads to a higher chance of preventing future bugs in case that one of those cases will be modified to not having a return. I agree that having a strategy is a better solution. I'd rather like to avoid introducing extra object for now though.